libs/ui/src/ListBox/ListBox.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react'
+
+import { ListBox } from './ListBox'
+
+describe('ListBox', () => {
+  it('should pass classnames properly', () => {
+    const { rerender } = render(<ListBox />)
+
+    const ul = screen.getByRole('list')
+
+    expect(ul).toHaveClass('_list_c59894')
```

Comment:
This id doesn't look like it's persistent, does it stay the same between initialisations?

Reply:
Removed it

Reply:
Removed it